designer/server/plugins/routes/api.ts

```diff
@@ -61,7 +61,7 @@ export const putFormWithId: ServerRoute = {
             request.payload
           );
 
-          throw new Error("Schema validation failed", error);
+          throw new Error("Schema validation failed, reason::" + error.message);
```

Comment:
```suggestion
          throw new Error("Schema validation failed, reason: " + error.message);
```
Minor suggestion but if we stick a space in here it'll be a little easier to read when whatever is contained in `error.message` is populated

Reply:
will make changes